appveyor.yml

```diff
@@ -6,7 +6,6 @@ environment:
   bundler_url: https://rubygems.org/downloads/bundler-1.9.9.gem
 
   matrix:
-    - ruby_version: "193"
```

Comment:
imho we should either add ruby_version 21 here or remove it from bundler above; what do you think?